lib/names.py

```diff
@@ -407,7 +407,7 @@ def public_dns(self, name):
         'cache_session': 'cache-session',   # Redis server for Django sessions.
         "cache": "cache",                   # Redis server to cache cuboids.
         "cache_state": "cache-state",
-        "cache_manager": "cachemanager",
+        "cachemanager": "cachemanager",
```

Comment:
These changes don't matter, as all of this code is commented out and was left in to help with merges when developing refactor

Reply:
I does matter,  if you look back at Slack - Deployment - Sept 30th.  If we don't make this change then we would need to do:
bastion.py cache_manager.hiderrt1.boss ssh
instead of cachemanager

Reply:
In `names.py` there are two copies of `AWSNames` the second copy is commented out and was left in `names.py` so that when merging changes from integration into refactor the merge happened easier. This second copy of `AWSNames` can be removed now as we are done doing development.